arch/x86/core/fatal.c

```diff
@@ -285,6 +285,10 @@ static void log_exception(uintptr_t vector, uintptr_t code)
 		LOG_ERR("Security exception");
 		break;
 	default:
+		/* Add an empty default with break, this is a defensive programming.
+		 * Static analysis tool won't raise a violation if default is empty
+		 * with a comment
+		 */
```

Comment:
Looking the code, wouldn't be better to log the unhandled exception ? Something like `LOG_ERR("Not handled excpetion (code 0x%lx)", code);`

Reply:
Yes, I did it.